app/main.py

```diff
@@ -1,3 +1,60 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> object:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return "Distance(km=" + str(self.km) + ")"
+
+    def __add__(self, other: None) -> object:
```

Comment:
Indicate in the type annotation that `other` could be `int`, `float`, or an instance of the `Distance` class. Change your return type annotation to indicate that you return an instance of the `Distance` class. Revise all your methods and change your type annotation to display correct incoming and outgoing types.